neo4j-jdbc-bolt/src/test/java/org/neo4j/jdbc/bolt/BoltNeo4jStatementIT.java

```diff
@@ -305,4 +305,81 @@ public static void tearDown(){
 		statement.close();
 	}
 
+	@Test public void testCallProcedure() throws SQLException {
+		connection.setAutoCommit(true);
+
+		Statement statement = connection.createStatement();
+		statement.execute("CALL db.schema.visualization");
+
+		ResultSet resultSet = statement.getResultSet();
+		resultSet.next();
+		Array arrayNodes = resultSet.getArray("nodes");
+		Object[] nodesArray = (Object[]) arrayNodes.getArray();
+		assertEquals(0, nodesArray.length);
+		Array arrayRels = resultSet.getArray("relationships");
+		Object[] relsArray = (Object[]) arrayRels.getArray();
+		assertEquals(0, relsArray.length);
+
+		statement.close();
+	}
+
+	@Test public void testSimpleReturn() throws SQLException {
+		connection.setAutoCommit(true);
+
+		Statement statement = connection.createStatement();
+		statement.execute("RETURN 1 AS id");
+
+		ResultSet resultSet = statement.getResultSet();
+		resultSet.next();
+		final long id = resultSet.getLong("id");
+		assertEquals(1L, id);
+
+		statement.close();
+	}
+
+	@Test public void testUnwindNope() throws SQLException {
+		connection.setAutoCommit(true);
+
+		Statement statement = connection.createStatement();
+		final boolean execute = statement.execute("UNWIND [] AS nope RETURN nope");
+		assertTrue(execute);
+
+		ResultSet resultSet = statement.getResultSet();
+		resultSet.next();
+		final Object nope = resultSet.getObject("nope");
+		assertNull(nope);
+
+		statement.close();
+	}
+
+	@Test public void testCreate() throws SQLException {
+		connection.setAutoCommit(true);
+
+		Statement statement = connection.createStatement();
+		final boolean execute = statement.execute("CREATE (n:NodeTest)");
+		assertFalse(execute);
```

Comment:
maybe worth testing that `statement.getResultSet()` returns `null`?

Reply:
done